app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Dotenv\Validator;
+use Illuminate\Auth\Events\Validated;
+use Illuminate\Contracts\Validation\Validator as ValidationValidator;
```

Comment:
remove all the extra `Validator` class and import right class here

Reply:
solved